mne/event.py

```diff
@@ -746,7 +746,7 @@ def make_fixed_length_events(raw, id, start=0, stop=None, duration=1.,
         A raw object to use the data from.
     id : int
         The id to use.
-    start : float
+    start : int | float
```

Comment:
I think I added this because `start` is often used (in the tests for example) as an int. The calculations later in the function cast it to int, so I just removed the int part from the docstring, but left the error when start is neither int nor float.